src/.nuget/Directory.Build.props

```diff
@@ -2,10 +2,12 @@
 <Project>
 
   <!-- Packaging projects (.pkgproj) are non-SDK-style, so they need to directly import Directory.Build.props -->
-  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.props))"
-          Condition="'$(MSBuildProjectExtension)' == '.pkgproj'" />
+  <Import Project="../Directory.Build.props" />
   <Import Project="packaging.props" />
 
+  <!-- Allow specialization of the packaging by creating a props file of the same name of the project immediately next to the project -->
+  <Import Condition="Exists('$(MSBuildProjectDirectory)/$(MSBuildProjectName).props')" Project="$(MSBuildProjectDirectory)/$(MSBuildProjectName).props"/>
```

Comment:
Did you have a chance to try renaming the Microsoft.NETCore.Native.props instead? I think it should work and it would be more in line with msbuild's existing system for customization.

Reply:
Yeah, that looks to work.